app/test/unit/git/remote-test.ts

```diff
@@ -61,6 +61,22 @@ describe('git/remote', () => {
       const remotes = await getRemotes(repository)
       expect(remotes).toHaveLength(0)
     })
+
+    it('returns promisor remote', async () => {
```

Comment:
Would you mind updating this PR to this test and removing the new fixture. The fixtures are more useful if there is repository scenario used across multiple tests. 

```
it('returns promisor remote', async () => {
      const repository = await setupEmptyRepository()

      // Add a remote
      const url = 'https://github.com/desktop/not-found.git'
      await GitProcess.exec(
        ['remote', 'add', 'hasBlobFilter', url],
        repository.path
      )

      // Fetch a remote and add a filter
      await GitProcess.exec(['fetch', '--filter=blob:none'], repository.path)

      // Shows that the new remote does have a filter
      const rawGetRemote = await GitProcess.exec(
        ['remote', '-v'],
        repository.path
      )
      expect(rawGetRemote.stdout).toContain(url + ' (fetch) [blob:none]')

      // Shows that the `getRemote` returns that remote
      const result = await getRemotes(repository)
      expect(result).toHaveLength(1)
      expect(result[0].name).toEqual('hasBlobFilter')
    })
```

Reply:
Yeah absolutely can do. Definitely seems better than having to add a new fixture.

Reply:
Updated! Much simpler indeed!